Modix/Services/Cat/CatService.cs

```diff
@@ -5,81 +5,68 @@
     using System.Net.Http;
     using System.Threading;
     using System.Threading.Tasks;
+    using Modix.Modules;
 
-    public class CatService
+    public interface ICatService
+    {
+        Task<string> HandleCat(CatModule.Media mediaType, CancellationToken token);
+    }
+
+    public class CatService : ICatService
     {
         private static readonly HttpClient Client = new HttpClient();
```

Comment:
I'd make this a private readonly member, and register the service as a singleton instead